ports/servoshell/panic_hook.rs

```diff
@@ -40,7 +41,19 @@ pub(crate) fn panic_hook(info: &PanicInfo) {
     drop(stderr);
 
     if opts::get().hard_fail && !opts::get().multiprocess {
-        std::process::exit(1);
+        // When we are exiting due to a hard-failure mode, we trigger a segfault so that crash
```

Comment:
Can we extract this change to a separate PR, since it's not related to fixing the original issue?